### ISSUE TITLE
[SUGGESTION]Make Steam Turbine can use IE Steam only.

### ISSUE THREAD

**Issue by Tollainmear** [FerroO2000/Immersive-Tech#51]
Hi, I found that Steam Turbine could also using steam which generated by other mods, such as TE Steam Dynamo, and the Steam Dynamo has a  amazing performance -  generating 27600mB steam only by burning a coal and 200mB water!

So that may totally break the balance, so I Strongly Recommend that please make Steam Turbine can use IE Steam only！(or make it be configuable might also be great)
😅

**Comment by wormzjl** [comment]
Yeah this can be a major exploit when used in a multimod environment...Hello power loop

**Comment by Tollainmear** [comment]
@FerroO2000 Hi~ Do you think it should be  a defect?

**Comment by wormzjl** [comment]
https://www.reddit.com/r/feedthebeast/comments/8uq2ii/the_most_efficient_power_i_have_seen_so_far?sort=confidence

**Comment by FerroO2000** [comment]
Yes, I must do something

**Comment by BigKid1973** [comment]
Another idea would be to balance this a little...
Bring the solar tower and the boiler about on par with the steam dynamo (or a few steam dinamos) and increase the intake of the turbine...
Or - provide all necessary options and add some comments in the cfg ...

**Comment by ranger135xp** [comment]
Any fix on this one?

**Comment by ranger135xp** [comment]
@FerroO2000 have you abandoned this mod?
I must say, that's pretty disappointing.
